docs/source/conf.py

```diff
@@ -39,27 +38,36 @@
 # -- Options for HTML output ----------------------------------------------
 # ref: http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output
 #
-html_theme = "jupyterhub_sphinx_theme"
+html_theme = "pydata_sphinx_theme"
 html_theme_options = {
+    "use_edit_page_button": True,
+    "github_url": "https://github.com/jupyterhub/binderhub",
+    "twitter_url": "https://twitter.com/mybinderteam",
     "icon_links": [
         {
-            "name": "GitHub",
-            "url": "https://github.com/jupyterhub/mybinder.org-deploy",
-            "icon": "fa-brands fa-github",
+            "name": "Discourse",
+            "url": "https://discourse.jupyter.org/",
+            "icon": "fa-brands fa-discourse",
+            "type": "fontawesome",
+        },
+        {
+            "name": "Team Compass",
+            "url": "https://jupyterhub-team-compass.readthedocs.io/en/latest/",
+            "icon": "fa-solid fa-compass",
+            "type": "fontawesome",
         },
     ],
-    "use_edit_page_button": True,
 }
 html_context = {
     "github_user": "jupyterhub",
     "github_repo": "mybinder.org-deploy",
-    "github_version": "HEAD",
+    "github_version": "main",
```

Comment:
```suggestion
    # Branch name is needed to ensure the `Edit on GitHub` button works
    "github_version": "main",
```
Added a comment so this doesn't get reverted in future